src/ViewInjection/ContentViewInjection.php

```diff
@@ -6,21 +6,17 @@
 
 use Yiisoft\Form\Widget\Field;
 use Yiisoft\Router\UrlGeneratorInterface;
-use Yiisoft\Router\UrlMatcherInterface;
 use Yiisoft\Yii\View\ContentParametersInjectionInterface;
 
 class ContentViewInjection implements ContentParametersInjectionInterface
 {
-    private UrlMatcherInterface $urlMatcher;
     private UrlGeneratorInterface $url;
     private Field $field;
 
     public function __construct(
-        UrlMatcherInterface $urlMatcher,
```

Comment:
Shouldn't we inject `CurrentRoute` here instead?

Reply:
No. Already injected in `commonParameters` https://github.com/yiisoft/demo/pull/331/files#diff-b59c295812dde5ba7714e8519aac18c6ec217ddb1313c1bdbc7ba5c466d5261bR16